tests/version-label.sh

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash
 # The purpose of this test is to ensure that the output of the "nodeos --version" command matches the version string defined by our CMake files
-# If the environment variable BUILDKITE_TAG is empty or unset, this test will echo success
```

Comment:
Do you remember, @arhag?

Reply:
Not sure. Maybe it is because the version was based on the tag at some point and so without the tag the version output would have been wrong? But that is just a complete guess. Also, I think the version info has not been based on tag for a while now (I think it uses CMake variables now).

Reply:
Thanks! I forgot about that. @williamblevins and I want to let it run for each build from now on.